src/contexts/wishlist.context.js

```diff
@@ -0,0 +1,31 @@
+import { createContext, useContext, useReducer } from "react";
+import { wishlistArr } from "../data/index.js";
+
+const WishListContext = createContext()
+
+export const WishListProvider = (props) => {
```

Comment:
You can directly pass children prop here. Like this `{children}`.

Reply:
made the changes. thanks.